docs/README.md

```diff
@@ -91,6 +91,8 @@ msbuild Datadog.Trace.proj /t:CreateHomeDirectory /p:Configuration=Release;Platf
 To build C# projects and NuGet packages only
 - [.NET 5.0 SDK](https://dotnet.microsoft.com/download/dotnet/5.0)
 - [.NET Core 3.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.1)
+- Optional: [.NET Core 2.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1) to test in .NET Core 2.1 locally.
+- Optional: [.NET Core 3.0 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.0) to test in .NET Core 3.0 locally.
```

Comment:
change order

Reply:
The first two are required, the bottom two optional, right?

Reply:
```suggestion
- Optional: [.NET Core 3.0 Runtime](https://dotnet.microsoft.com/download/dotnet-core/3.0) to test in .NET Core 3.0 locally.
- Optional: [.NET Core 2.1 Runtime](https://dotnet.microsoft.com/download/dotnet-core/2.1) to test in .NET Core 2.1 locally.
```

Reply:
Accepted your suggestion and did in other place for consistency.